src/components/Caption/index.tsx

```diff
@@ -7,7 +7,7 @@ export interface CaptionProps {
   title: JSX.Element;
   className?: string;
   color?: "green" | "sky" | "blue";
-  textSize?: "small" | "medium" | "costume";
+  textSize?: "small" | "medium" | "large" | "costume";
```

Comment:
Well.. if you run VSCode's find and replace you can get all of these in one go lol